WMFComponents/Sources/WMFComponents/Components/Article Tabs/WMFArticleTabsViewModel.swift

```diff
@@ -12,6 +12,7 @@ public class WMFArticleTabsViewModel: NSObject, ObservableObject {
     // articleTab should NEVER be empty - take care of logic of inserting main page in datacontroller/viewcontroller
     @Published var articleTabs: [ArticleTab]
     @Published var shouldShowCloseButton: Bool
+    @Published var hasMultipleTabs: Bool = false
```

Comment:
Can we just make this a computed variable instead?

```
var hasMultipleTabs {
    return articleTabs.count >= 2
}
```
 
This way we don't have to worry about keeping hasMultipleTabs in sync every time a tab is opened or closed.